.github/workflows/sec-opengrep.yml

```diff
@@ -0,0 +1,44 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+name: Security - Opengrep
+
+on:
+  workflow_call:
+
+permissions:
+  contents: read
+  security-events: write
+
+env:
+  # Release version of Opengrep
+  # https://github.com/opengrep/opengrep/releases
+  RELEASE_VERSION: v1.0.0-alpha.13
```

Comment:
I presume they will release an action wrapper at some point, or we can grab latest release via script

Reply:
I'll send in a PR if not